.devcontainer/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Attach Citus (devcontainer)",
+            "type": "cppdbg",
+            "request": "attach",
+            "processId": "${command:pickProcess}",
```

Comment:
I also find below config useful here. SIGUSR1 is commonly used for communication between processes and postmaster. By default gdb intercepts the signal and stop the process. Instead I let gdb to pass the signal to process.

```
"setupCommands": [
        {
          "text": "handle SIGUSR1 noprint nostop pass",
          "description": "let gdb pass SIGUSR1 to program",
          "ignoreFailures": true
        }
],
```

I had problems with SIGUSR1 but we may want to add `handle all nostop` to pass all signals.

Reply:
That is awesome feedback, I have only recently started using vscode, so haven't run into this yet.

Can you remind me wat SIGURS1 in the context of postgres is used for? This so that I can check how the workflow is now, and how this change changes it? So to make sure I correctly apply this change.

Reply:
I did a quick search on https://sourceware.org/gdb/onlinedocs/gdb/Signals.html. By default, gdb passes all signals except SIGTRAP to the process. But it stops the process for only some of the signals. SIGUSR1 is one of those signals that it stops for. I added the line to prevent gdb to stop the execution when SIGUSR1 comes. Same may apply to some other signals. It can be tedious to continue the execution manually for them.

You can try `kill -s SIGUSR1 <pid>` while debugging the process to see how it stops.